generators/entity/templates/src/main/resources/config/liquibase/changelog/_added_entity_constraints.xml

```diff
@@ -12,6 +12,14 @@
             relationshipName = relationships[idx].relationshipName,
             ownerSide = relationships[idx].ownerSide,
             otherEntityName = relationships[idx].otherEntityName;
+            if (otherEntityName === 'user') {
+                otherEntityTableName = jhiPrefix + '_user';
+            } else {
+                otherEntityTableName = getEntityProperty(otherEntityName, 'entityTableName');
+                if (otherEntityTableName === undefined) {
```

Comment:
you are returning null always from the method so this will not be undefined. plz change to `if (otherEntityTableName) {` so that null, undefined and empty is checked